templates/opp-single-form.php

```diff
@@ -4,8 +4,14 @@
  *
  * This template is displayed immediately after the_content() is called within your theme file.
  * To adjust this template copy it into your current theme within a folder called "wivm".
+ *
```

Comment:
Great idea with this note so that the user knows the importance of retaining it if they choose to use a custom template.